staging/src/k8s.io/kubectl/pkg/cmd/label/label.go

```diff
@@ -392,6 +393,11 @@ func (o *LabelOptions) RunLabel() error {
 		if err != nil {
 			return err
 		}
+
+		if operationCounter > 0 && *o.PrintFlags.OutputFormat == "yaml" {
```

Comment:
Basically `YamlPrinter` is responsible for adding separators https://github.com/kubernetes/kubernetes/blob/e4fca6469022309bc0ace863fce73054a0219464/staging/src/k8s.io/cli-runtime/pkg/printers/yaml.go#L48 and I'd prefer not to add this into `label` command directly. 

Because if there is any problem in `YamlPrinter`, other commands are also being affected already and we need to fix them all.

Reply:
Completely agree. I opened this PR anyway to see what would be your inputs.
I'll explore a little bit further and see if I can change the printer's operation after it has been instantiated. Do you have any other suggestion?

Reply:
I tried making printCount from YAMLPrinter as a global class variable. Not the most elegant solution, but it does work in the given scenario. Would you mind taking another look? Thanks!!